docs/OnnxReleases.md

```diff
@@ -1,53 +1,43 @@
-* Make sure all the git submodules are updated
+* Make sure onnx/wheel_builder is working
+* Pick a version number and release tag for the new release through mutual consent – Gitter Room for Releases (https://gitter.im/onnx/Releases)
+* Create a PR to update VERSION_NUMBER (https://github.com/onnx/onnx/blob/master/VERSION_NUMBER) and versioning doc (https://github.com/onnx/onnx/blob/master/docs/Versioning.md)
+* Create a release branch, for ex rel-1.7.0
+* Create a PR against release branch to update VERSION_NUMBER for release verification in TestPypi. Make sure CI is clean.
+* After PR merged, download Windows wheels from git repo Actions, https://github.com/onnx/onnx/actions, and use the commit ID to update .travis in wheel-builder
+* Create a branch pypi-test in onnx/wheel-builder repo and a PR to trigger CI to upload release candidate wheels for Linux and OSX to TestPypi
+* Upload Windows wheels to TestPypi ``python -m twine upload --verbose *.whl --repository-url https://test.pypi.org/legacy/ -u onnx -p pwd``
+* Build ONNX source distribution
     * ``git submodule update –init``
-* Make sure the git checkout is clean –
     * Do ``git clean -nxd`` and make sure that none of the auto-generated header files *like* the following are not present.
         * onnx/onnx-operators.pb.cc
         * onnx/onnx-operator.pb.h
         * onnx/onnx.pb.cc
         * onnx/onnx.pb.h
     * If they are present run ``git clean -ixd`` and remove those files from your local branch
-* Do ``python setup.py sdist`` to generate the source distribution.
-* Put the following content into ``~/.pypirc`` file:
-```
-[distutils]
-index-servers =
-  pypi
-  pypitest
- 
-[pypi]
-username=<username>
-password=<password>
- 
-[pypitest]
-repository=https://test.pypi.org/legacy/
-username=<username>
-password=<password>
-```
-* Do ``twine upload dist/* -r pypitest`` to upload it to the test instance of pypi.
-* After uploading to pypitest, you can test the source distribution by doing ``pip install –index-url https://test.pypi.org/simple/ onnx`` in a new environment. Test this installation with different environments and versions of protobuf binaries.
-* *NOTE - Once a distribution is uploaded to pypi, you cannot overwrite it on the same pypi instance.*
-* Once completely verified do, ``twine upload dist/*``  to upload to the official pypi.
-* *Conda - *
+    * ``python setup.py sdist``
+* Upload source distribution to TestPypi ``python -m twine upload --verbose dist/* -r --repository-url https://test.pypi.org/legacy/ -u onnx -p pwd``
+* After uploading to TestPypi, you can test the source distribution by doing ``pip install –index-url https://test.pypi.org/simple/ onnx`` in a new environment. Test this installation with different environments and versions of protobuf binaries.
+* Verify the release candidate in TestPypi by release test team. The communication is via gitter and an github issue.
```

Comment:
just curious: Is this created for every release? I didn't know there was a release test team in place.

Reply:
This is again similar to my previous comment. I just made the release candidates available, announced it in infra gitter/github issue, and a few people would start the verification and eventually report status. That's what we need to clarify in release WG and for 1.8.